examples/camunda-8.6-irsa/procedure/create-setup-db-secret.sh

```diff
@@ -0,0 +1,12 @@
+kubectl create secret generic setup-db-secret --namespace camunda \
+  --from-literal=AURORA_ENDPOINT="$AURORA_ENDPOINT" \
+  --from-literal=AURORA_PORT="$AURORA_PORT" \
+  --from-literal=AURORA_DB_NAME="$AURORA_DB_NAME" \
```

Comment:
for me `AURORA_DB_NAME` is not defined by this point.
It was also not exported anywhere prior in the documentation.